code/modules/food_and_drink/pizza.dm

```diff
@@ -546,8 +546,8 @@
 	desc = "A cheesy pizza slice topped with fresh picked mushrooms."
 	initial_reagents = list("bread" = 2, "juice_tomato" = 2, "cheese" = 1, "space_fungus" = 4)
 
-/obj/item/reagent_containers/food/snacks/pizzaslice/standard
-	name = "slice of fresh basic pizza"
+/obj/item/reagent_containers/food/snacks/pizzaslice/vendor
+	name = "slice of basic pizza"
 	desc = "A slice of base vendor pizza (you shouldn't see this)."
```

Comment:
if they shouldnt see it then why isnt it abstract HUH

Reply:
cant be because abstract tosses itself up into the parent for some reason, which causes a runtime with additional runtime checking on

Reply:
it intentionally does that, but why would it cause a runtime

Reply:
because the parent (pizzaslice) is not uh intended to be abstract because it is created when slicing a bespoke pizza which i can repath into like pizzaslice/bespoke if you want and abstract pizzaslice, pizzaslice/vendor, and pizzaslice/standard

Reply:
i think that would make more sense